app/controllers/user_invites_controller.rb

```diff
@@ -1,6 +1,8 @@
 class UserInvitesController < ApplicationController
+  load_and_authorize_resource instance_name: :invite
+
   def index
-    @invites = UserInvite.acceptable
+    @invites = @invites.acceptable
```

Comment:
Can you help me understand where this instance variable is coming from?

Reply:
The `@invites` instance variable is, amazingly, coming from the call to `load_and_authorize_resource` added above. It uses the name of the controller by default (e.g. `@user_invites`), but in this case I've overridden it to use `@invite[s]`.